packages/gitbook/src/components/DocumentView/Block.tsx

```diff
@@ -116,17 +115,7 @@ export function Block<T extends DocumentBlock>(props: BlockProps<T>) {
         }
     })();
 
-    if (!isEstimatedOffscreen || context.wrapBlocksInSuspense === false) {
-        // When blocks are estimated to be on the initial viewport, we render them immediately
-        // to avoid a flash of a loading skeleton.
-        return content;
-    }
```

Comment:
Why do you remove this?

Isn't it useful in the dynamic route case?

Reply:
It makes sense, it was needed when we were streaming the page, and even a paragraph could suspend as it has links in it.

But now dynamic pages are a edge-case and as long as we have a suspense boundary over the entire docs, it's fine!